scripts/remove_v_5_1.py

```diff
@@ -0,0 +1,294 @@
+import os
+import sys
+import shlex
+import socket
+import logging
+import argparse
+import subprocess
+from io import StringIO
+from os.path import join
+from tempfile import mkstemp
+
+from ruamel.yaml import YAML
+from ruamel.yaml.error import YAMLError
+
+INITIAL_INSTALL_DIR = '/etc/cloudify/.installed'
+INSTALLED_COMPONENTS_FILE = join(INITIAL_INSTALL_DIR, 'components.yaml')
+INSTALLED_PACKAGES = join(INITIAL_INSTALL_DIR, 'packages.yaml')
+
+logger = logging.getLogger(socket.gethostname())
+
+SOURCES = {
+    'manager': ['cloudify-management-worker', 'cloudify-rest-service',
+                'cloudify-cli', 'cloudify-manager-ip-setter', 'nginx',
+                'python-psycopg2', 'postgresql95', 'cloudify-agents',
+                'patch', 'nodejs', 'cloudify-stage'],
+    'manager_premium': ['cloudify-premium', 'cloudify-composer'],
+    'manager_cluster': ['haproxy'],
+    'db': ['postgresql95', 'postgresql95-server', 'postgresql95-contrib',
+           'libxslt'],
+    'db_cluster': ['libestr', 'libfastjson', 'rsyslog', 'etcd', 'patroni'],
+    'queue': ['rabbitmq-server-3.8.4', 'cloudify-rabbitmq'],
+    'queue_cluster': [],
+    'prometheus': ['prometheus', 'node_exporter', 'blackbox_exporter',
+                   'postgres_exporter'],
+    'prometheus_cluster': ['nginx'],
+    'haveged': ['haveged']
+}
+
+
+class ProcessExecutionError(BaseException):
+    def __init__(self, message, return_code=None):
+        self.return_code = return_code
+        super(ProcessExecutionError, self).__init__(message)
+
+
+def setup_logger(verbose):
+    global logger
```

Comment:
this line does nothing

Reply:
Thanks, removed. But just for my understanding, don't we need to declare `global` in python in case we use a variable from outside the function?

Reply:
Only if you mutate it.

Reply:
Thanks! I'll remember this.